src/Xamarin.Android.Build.Tasks/Tasks/AndroidCreateDebugKey.cs

```diff
@@ -27,16 +27,31 @@ protected override CommandLineBuilder CreateCommandLine ()
 			return cmd;
 		}
 
+		bool warnExitCode = false;
+
 		protected override void LogFromStandardError (string text)
 		{
 			// Downgrade error message to warning:
 			// error: java.lang.Exception: Key pair not generated, alias <androiddebugkey> already exists
 			if (text.Contains ($"alias <{KeyAlias}> already exists")) {
 				text = text.Replace ("error:", "warning:");
+				warnExitCode = true;
 			}
 
 			base.LogFromStandardError (text);
 		}
+
+		protected override bool HandleTaskExecutionErrors ()
+		{
+			// Warn for the exit code if we get:
+			// error: java.lang.Exception: Key pair not generated, alias <androiddebugkey> already exists
+			if (warnExitCode && !Log.HasLoggedErrors) {
+				Log.LogWarningFromResources ("ToolTask.ToolCommandFailed", ToolExe, ExitCode);
```

Comment:
Alas, this resource apparently doesn't exist:

https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=4432778&view=ms.vss-test-web.build-test-results-tab&runId=18631396&resultId=100000&paneView=attachments

```
System.InvalidOperationException: The "AndroidCreateDebugKey" task has not registered its resources. In order to use the "TaskLoggingHelper.FormatResourceString()" method this task needs to register its resources either during construction, or via the "TaskResources" property.
  at Microsoft.Build.Shared.ErrorUtilities.ThrowInvalidOperation (System.String resourceName, System.Object[] args)
  at Microsoft.Build.Shared.ErrorUtilities.VerifyThrowInvalidOperation (System.Boolean condition, System.String resourceName, System.Object arg0)
  at Microsoft.Build.Utilities.TaskLoggingHelper.FormatResourceString (System.String resourceName, System.Object[] args)
  at Microsoft.Build.Utilities.TaskLoggingHelper.LogWarningFromResources (System.String subcategoryResourceName, System.String warningCode, System.String helpKeyword, System.String file, System.Int32 lineNumber, System.Int32 columnNumber, System.Int32 endLineNumber, System.Int32 endColumnNumber, System.String messageResourceName, System.Object[] messageArgs)
  at Microsoft.Build.Utilities.TaskLoggingHelper.LogWarningFromResources (System.String messageResourceName, System.Object[] messageArgs)
  at (wrapper remoting-invoke-with-check) Microsoft.Build.Utilities.TaskLoggingHelper.LogWarningFromResources(string,object[])
  at Xamarin.Android.Tasks.AndroidCreateDebugKey.HandleTaskExecutionErrors ()
  at Microsoft.Build.Utilities.ToolTask.Execute ()
```

Reply:
Trying locally this seems to fix it:
```
TaskResources = new ResourceManager ("Microsoft.Build.Utilities.Core.Strings", typeof (ToolTask).Assembly);
```